### ISSUE TITLE
Additional Problem status type: 'in-progress' needs MS code changes

### ISSUE THREAD

**Issue by FrankSylvia** [center-for-knowledge-communication/mathspring#4]
Recently we incompletely implemented a third status type: in-progress. This is the new default (starting) status for problems. 'Testable' will now mean that the problem authoring is complete and its ready for testing. But we ran into problems. First, MS should show in-progress (and testable) problems to test-users and it does not.

Second, "there are difficulties with previewing in-progress from the MsAdmin" (Danielle).
Basically it means that problems that are "in-progress" should show up in the following screen (see attached).

**Comment by FrankSylvia** [comment]
<img width="855" alt="problem_status" src="https://user-images.githubusercontent.com/46008380/66714852-2abfe680-ed8a-11e9-9a4f-3760a79f3234.png">

**Comment by FrankSylvia** [comment]
Moved from previous repo

**Comment by FrankSylvia** [comment]
Released to QA

**Comment by IvonArroyo** [comment]
This one is ready to be closed, how to close an issue?

**Comment by ankitrgupta1** [comment]
Reopened the issue as it is yet to go to prod. No action needed as of now.
